scripts/db_migrator.py

```diff
@@ -91,7 +91,9 @@ def __init__(self, namespace, socket=None):
         self.asic_type = version_info.get('asic_type')
         if not self.asic_type:
             log.log_error("ASIC type information not obtained. DB migration will not be reliable")
-        self.hwsku = device_info.get_hwsku()
+
+        # TODO: Update get_hwsku API to take in a custom configDB object
```

Comment:
Why is this TODO left? How will custom configDB help unblock the busy wait of get_hwsku?

Reply:
When get_hwsku() is used, it calls get_localhost_info and it calls ConfigDBConnector.connect() which busy waits. 

I'll remove the TODO comment